chart/env/prod.yaml

```diff
@@ -395,7 +395,7 @@ search:
       memory: "6500Mi"
     limits:
       cpu: 16
-      memory: "9500Mi"
+      memory: "28610Mi"
```

Comment:
Note that the machines (nodes) are defined here: 

https://github.com/huggingface/infra/blob/6ebd3fde4c220f8396de7cc7a884b649a0eb821d/projects/hub/00-aws/variables/hub-aws-prod-us-east-1.tfvars#L187-L196

Currently using m6a.4xlarge (https://aws.amazon.com/ec2/instance-types/m6a/); ie: each node only has 64GiB RAM, we might prefer bigger ones, even if it would work with two pods per node with this change.

Reply:
Also:

1. we should set the `requests.memory` and `limits.memory` parameters to the same value, because otherwise it will break quickly
2. with 64GiB and two pods per node, we can increase this value to 30Gi or 31Gi (we still have to let some RAM for the host machine)